src/obj.zig

```diff
@@ -295,11 +295,7 @@ pub const ObjFiber = struct {
     };
 };
 
-pub const pcre_struct = switch (builtin.os.tag) {
```

Comment:
This maybe depends on you're pcre install? it tested this on ubuntu-latest without any issue. Anyway you should not remove the macos, etc. case

Reply:
I use Arch Linux. I think `pcre_*` should be used with `pcre` type.

```
PCRE_EXP_DECL int  pcre_exec(const pcre *, const pcre_extra *, PCRE_SPTR,
                   int, int, int, int *, int);
PCRE_EXP_DECL int  pcre16_exec(const pcre16 *, const pcre16_extra *,
                   PCRE_SPTR16, int, int, int, int *, int);
PCRE_EXP_DECL int  pcre32_exec(const pcre32 *, const pcre32_extra *,
                   PCRE_SPTR32, int, int, int, int *, int);
```

Reply:
are you sure you not using pcre2 and not pcre?

Reply:
It's not pcre2.